mirar/database/user/postgres_admin.py

```diff
@@ -34,9 +50,11 @@ def create_new_user(self, new_db_user: str, new_password: str):
         :return: None
         """
         engine = get_engine(
-            db_name="postgres",
+            db_name=self.db_name,
```

Comment:
I'm trying to understand what the logic will be here. The intention was to create a user before the database is actually created, hence `postgres`, so that the user will own the database of name `self.db_name`. Will that still work under this scheme?

Reply:
Nope, you are correct. I've changed this back to "postgres" now.